tools/make-v8.sh

```diff
@@ -20,7 +20,7 @@ if [[ "$ARCH" == "s390x" ]] || [[ "$ARCH" == "ppc64le" ]]; then
 fi
 if [[ "$ARCH" == "s390x" ]]; then
   ln -s $CXX_PATH "$BUILD_TOOLS/g++"
-  ln -s $CXX_PATH "$BUILD_TOOLS/gcc"
+  ln -s $CC_PATH "$BUILD_TOOLS/gcc"
```

Comment:
> Current CC flag points to g++ instead of gcc

Looking at the changes I guess you actually meant: Current CXX flag points to gcc instead of g++?

Reply:
Sorry I meant gcc is pointing to CXX (g++), I have fixed it above.

Reply:
`CC_PATH` isn't defined anywhere.